src/main/java/org/scalameter/japi/annotation/scopes.java

```diff
@@ -0,0 +1,33 @@
+package org.scalameter.japi.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/** Allows to set custom contexts for scopes.
+ *
+ *  It should consists of @scopeCtx that each points to a single scope and context.
+ *
+ *  Note that it should be placed over a class
+ *  that extends `JBench` and contains benchmark methods.
+ *  {{{
+ *    @scopes({
+ *      // sets context for the scope "arrays.forloops" pointed by a `ctxVariable`
+ *      @scopeCtx(scope = "arrays.forloops", ctx = "ctxVariable")
+ *    )}
+ *    class ArrayBenchmark extends JBench.Micro {
+ *      // @...
```

Comment:
Maybe we could remove this line to avoid confusing the reader, and instead add a ctxVariable member.